packages/expo-cellular/src/Cellular.ts

```diff
@@ -1,12 +1,14 @@
 import { UnavailabilityError } from '@unimodules/core';
 
-import { CellularGeneration } from './Cellular.types';
+import { CellularGeneration, CellularInfo } from './Cellular.types';
 import ExpoCellular from './ExpoCellular';
 
 export { CellularGeneration };
 
 // @needsAudit
 /**
+ * @deprecated Deprecated field, use [`getCurrentCellularInfoAsync()`](#getcurrentcellularinfoasync) instead.
+ *
```

Comment:
@Simek Could you please have a look here? It looks like `@deprecated` doesn't detect the end of the line and docs renders that way:

<img width="730" alt="Screenshot 2021-07-22 at 6 06 51 PM" src="https://user-images.githubusercontent.com/9566628/126672250-799a6e5e-8e6d-4114-aef6-747a1ffc5e0f.png">

Reply:
Probably if you add an empty line after the `@deprecated` message in the code comment the description will be rendered properly.

Reply:
Hmm, looking and JSON file it looks like it's an extraction bug, so there might be the issue with the tool we use underneath (TypeDoc). Can you try, as a workaround for now, move the `@deprecated` paragraph below the description in the code comment and separate it by an empty line?

The other issue I can see in the screenshot is that the Markdown is not rendered inside the deprecation note block, which might need an fix in docs component. I will take care of that.

Reply:
@M1ST4KE If you update the PR branch, deprecated Markdown rendering issue should be fixed. Can you confirm? 🙂